landing_page/views.py

```diff
@@ -7,3 +9,12 @@ def index(request):
 
 
 # Add other views here
+def search_movies(request):
+    query = request.GET.get("query")
+    if query:
+        movies = Movie.objects.filter(title__icontains=query)
+    else:
+        movies = Movie.objects.all()
```

Comment:
Instead of returning all I think it would be better to return nothing

Reply:
yes! great thought.

Reply:
No movies are returned if nothing is found